doc/engineering-reference/src/climate-sky-and-solar-shading-calculations/climate-calculations.tex

```diff
@@ -44,86 +44,106 @@ \subsection{EnergyPlus Design Day Temperature Calculations}\label{energyplus-des
-Temperature\(_{drybulb}\) = drybulb temperature \{K\}
+$T_{sky}$ = Effective mean sky temperature, or sky radiative temperature, \si{\kelvin}.
+\newline
 
-By default, the clear sky emissivity is calculated by the Clark \& Allen (1978) model as
+Over the years authors have proposed a fictitious quantity called the Sky Emissivity ($\epsilon_{sky}$) such that the following energy balance is satisfied:
 
 \begin{equation}
-Sky_{clear-sky-emissivity} = 0.787 + 0.764\cdot \ln \frac {T_{dewpoint}}{T_{Kelvin}}
+IR_H = \epsilon_{sky} \cdot \sigma \cdot T_{db}^4
 \end{equation}
 
-where, T\(_{dewpoint}\) = dewpoint temperature \{K\}.
+$\epsilon_{sky}$ = sky emissivity
 
-Alternatively, the clear sky emissivity can also be calculated using different models set by the user from options using the \textbf{WeatherProperty:SkyTemperature} object, including the calibrated version of the Berdahl and Martin, Brunt and Idso models (2017), listed as follows.
-
-Calibrated Berdahl and Martin model:
-
-\begin{equation}
-Sky_{clear-sky-emissivity} = 0.711 + 0.56\cdot(T_{dewpoint} / 100) + 0.73\cdot{T_{dewpoint} / 100}^2.
-\end{equation}
+$T_{db}$ = drybulb temperature, \si{\kelvin}
+\newline
 
-Calibrated Brunt model:
+Four correlations for $\epsilon_{sky}$ under clear-sky conditions, proposed by four sets of authors, are available in EnergyPlus. The default correlation is from Clark \& Allen (1978):
 
-\begin{equation}
-Sky_{clear-sky-emissivity} = 0.52 + 0.065\cdot(P_{water-vapor})^{0.5},
-\end{equation}
+\begin{table}[hbtp]
+\centering
+\begin{tabular}{cl}
+\textbf{Author}   & \epsilon_{sky, clear} \\ \\
+Clark \& Allen    & $= 0.787 + 0.764 \ln\left(T_{dp}/273\right)$ \\ \\
+Martin \& Berdahl & $= 0.711 + 0.56 \left(T_{dp}/100\right) + 0.73  \left(T_{dp} / 100\right)^2$ \\ \\
+Brunt             & $= 0.52 + 0.065 \left(P_{wv}\right)^{0.5}$ \\ \\
+Idso              & $= 0.70 + 5.95\times10^{-5} \left(P_{wv}\right) e^{1500/T_{db}} $
+\end{tabular}
+\end{table}
 
-where, P\(_{water-vapor}\) = partial water vapor pressure \{hPa\}.
+\noindent where:
+\newline
 
-Calibrated Idso model:
+$\epsilon_{sky, clear}$ = $\epsilon_{sky}$ under clear-sky conditions
+\newline
 
-\begin{equation}
-Sky_{clear-sky-emissivity} = 0.70 + 5.95\cdot10E-5\cdot(P_{water-vapor} \cdot exp(1500/T_{drybulb})).
-\end{equation}
+$T_{dp}$ = dewpoint temperature, in \si{\kelvin}
+\newline
 
+$P_{wv}$ = partial pressor of water vapor, in \si{\hecto\pascal}
+\newline
 
-Then the sky emissivity is given by:
+The clear sky emissivity is modified for partially-cloudy conditions using the correlation from Walton (1983) which uses the opaque cloud cover fraction:
 
 \begin{equation}
-Sky_{emissivity} = Sky_{clear-sky-emissivity} \cdot \left( {1. + .0224N - .0035{N^2} + .00028{N^3}} \right)
+\epsilon_{sky} = \epsilon_{sky, clear} \left(1 + 0.0224  N - 0.0035  N^2 + 0.00028  N^3\right)
 \end{equation}
 
-where, N = opaque sky cover \{tenths\}
+\noindent where:
+\newline
+
+$N$ = opaque sky cover, in tenths.
```

Comment:
tenths?

Reply:
Yeah, that is a little weird now that I stop to think about it. "Fraction"? I think the intent was just a value from 0 to 1.

Reply:
It could be intentional/correct. Sky cover values are indeed stored in tenths in the EPW file. See 
https://bigladdersoftware.com/epx/docs/9-2/auxiliary-programs/epw-csv-format-in-out.html#data-records-csv:

`... TotSkyCvr {.1},OpaqSkyCvr {.1} ...`

Reply:
Fraction.  I'm not going to start to ask for design flow rate in "Thousandths"

Reply:
No, a lot of the weather data world is integer-based from way back when, so the values for sky cover range from 0-10 in tenths in the epw and that's how it's used in WeatherManager.

Reply:
And aerosol_optical_depth is in thousandths, values from 0-1000. Auxiliary Programs doc, p. 26.

Reply:
But it looks like it is declared as a [real](https://github.com/NREL/EnergyPlus/blob/bc433498e9decc526a1ccbee41f647d704689827/src/EnergyPlus/WeatherManager.cc#L2791), then [used](https://github.com/NREL/EnergyPlus/blob/bc433498e9decc526a1ccbee41f647d704689827/src/EnergyPlus/WeatherManager.cc#L3503) as a real [argument](https://github.com/NREL/EnergyPlus/blob/bc433498e9decc526a1ccbee41f647d704689827/src/EnergyPlus/WeatherManager.cc#L3754) in the sky emissivity function and applied as a continuous independent variable in the [correlation](https://github.com/NREL/EnergyPlus/blob/bc433498e9decc526a1ccbee41f647d704689827/src/EnergyPlus/WeatherManager.cc#L3777).  Though maybe I'm missing something.

If tenths really is preferred for historical reasons, then no problem.

Reply:
Oh sure, the values all probably get used as reals, but if you go back to the original weather data, it's all coming in as integers. One could argue that the variable should be an integer, but it likely is getting interpolated for sub-hourly time steps and has non-integer values then.

Reply:
> And aerosol_optical_depth is in thousandths, values from 0-1000. Auxiliary Programs doc, p. 26.

Ew.

Reply:
Yeah, it's best to just leave all the weather data to the weather experts and not look too closely . . . no offense, DC and LKL!

Reply:
:+1: I'm done :+1: